src/dune_rules/install_rules.ml

```diff
@@ -371,7 +371,7 @@ end = struct
             stanza_to_entries ~sctx ~dir ~scope ~expander stanza
           in
           named_entries :: acc)
-      |> Memo.Build.parallel_map ~f:Fun.id
+      |> Memo.Build.all
```

Comment:
Are you aware that `all` is sequential? Perhaps, you want `all_concurrently` here (at least if that's just a refactoring).

Reply:
Yes, I'm aware. I don't know enough about this code to know if concurrently will speed it up or just add overhead. My hunch would be that it would be just overhead.

Reply:
Just to clarify: it's not just about speeding things up. Without looking into the code, it's impossible to tell if the sequential version is not going to deadlock because the concurrent threads might actually interact somehow via Ivars.

Reply:
I'm confident there's no deadlock. All we are doing is loading rules and evaluating enabled_if. Besides, if there was a deadlock, we'd see it by now :)

By the way, if you think this is better done concurrently I can do the change. It's not something I did originally because I wasn't sure it would improve things.

Reply:
No, no, I'd prefer to use concurrency only when needed. As you say, it's unnecessary in most cases.

I just wanted to make sure we're on the same page regarding possible consequences of this change.